addons/base_import/tests/test_csv_magic.py

```diff
@@ -150,3 +150,41 @@ def test_parse_float(self):
             ['1.62', '-1.62', '+1.62', '+1.62', '-1.62',
              '1234567.89', '1234567.89']
         )
+
+
+class TestLanguageDetection(ImportCase):
+    def test_preview_language_detection(self):
+        """ Preview language warning test. The language warning informs the user that the imported file is in the wrong
+        language if the detected language of the file is different from the user language interface.
+        """
+        ResLang = self.env['res.lang']
+        french_display_name = 'French (BE) / Français (BE)'
+        ResLang.search([('code', '=', 'fr_BE'), ('active', '!=', 1)]).write({'active': 1, 'name': french_display_name})
```

Comment:
(do not use number on boolean field)
```suggestion
        ResLang.search([('code', '=', 'fr_BE'), ('active', '!=', True)]).write({'active': 1, 'name': french_display_name})
```